src/JsRoot/GraphData.tsx

```diff
@@ -17,7 +18,7 @@ export type pageData = {
 interface IPage {
 	filterRef?: FilterJSON;
 	name?: string;
-	dimensions: number;
+	dimensions: number | string;
```

Comment:
I was thinking of simplifying the design here. 
@ostatni5 what do you think of keeping the dimension as integer.
Then is dimensions == 3 then we present to the user results in form the same as `ResultsCT.tsx` ?
In fact results from CT scan are basically the same as of any 3-D scorer.

Reply:
So if the results are 3D will it always be a link?

Reply:
Yes